### PR TITLE
Get personal relationships for a person

### PR DESCRIPTION
**Pull request by LBHTKarki**
## Link to JIRA ticket

[JIRA ticket](https://hackney.atlassian.net/secure/RapidBoard.jspa?rapidView=61&projectKey=SCT&modal=detail&selectedIssue=SCT-203) - Create new service API GET endpoint

### *What is the problem we're trying to solve*

Add personal relationships end point that can be used by the front end to fetch relationship data for a person by using their id.

### *What changes have we introduced*

Add support for personal relationships. Data is fetched from the Social Care Platform API and includes the following personal relationships:

- Children
- Parents
- Siblings
- Other

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs [hunk 1]
```diff
@@ -1408,6 +1408,38 @@ public void UpdatePersonCreatesCorrectCaseHistoryNoteByCallingProcessDataGateway
             result["is_imported"].Should().Be(false);
         }
 
+        [Test]
+        public void GetPersonsByListOfIdsReturnsCorrectPersonRecordsBasedOnGivenIds()
```
Comment (chubberlisk) [inline SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs:1412]: Maybe a test for documenting what happens when no IDs match?

**Review by chubberlisk — APPROVED** [review]
Looks good to me!

File: SocialCareCaseViewerApi/V1/Controllers/SocialCareCaseViewerApiController.cs [hunk 2]
```diff
@@ -455,5 +457,27 @@ public IActionResult PatchWarningNote([FromBody] PatchWarningNoteRequest request
                 return NotFound(e.Message);
             }
         }
+
+        /// <summary>
+        /// Get a list of relationships by person id
+        /// </summary>
+        /// <param name="request"></param>
+        /// <response code="200">Successful request. Relationships returned</response>
+        /// <response code="404">Person not fouund</response>
```
[the remark this answers is not in the document]
Comment (LBHTKarki) [reply SocialCareCaseViewerApi/V1/Controllers/SocialCareCaseViewerApiController.cs]: fixed

File: SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs (hunk 1 above)
Comment (LBHTKarki) [reply SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs:1412]: Good shout, I've added a test to cover that.

**Pull request merged**